doc/sphinx/language/extensions/arguments-command.rst

```diff
@@ -50,51 +50,58 @@ Setting properties of a function's arguments
    it's not in a section.  Applying :attr:`global` within a section extends the
    effect outside the current sections and current module in which the command appears.
 
-      `/`
-         the function will be unfolded only if it's applied to at least the
-         arguments appearing before the `/`.  See :ref:`Args_effect_on_unfolding`.
+      :n:`@name {* % @scope }`
+         a *formal parameter* of the function :n:`@reference` (i.e.
+         the parameter name used in the function definition).  Unless `rename` is specified,
+         the list of :n:`@name`\s must be a prefix of the formal parameters, including all implicit
+         arguments.  `_` can be used to skip over a formal parameter.
+         This construct declares :n:`@name` as
+         non-implicit if `clear implicits` is specified or any
+         other :n:`@name` in the :cmd:`Arguments` command is declared implicit.
+         :token:`scope` can be either scope names or their delimiting
+         keys. When multiple scopes are present, notations are interpreted in the
+         leftmost scope containing them. See :ref:`binding_to_scope` for details.
 
-         .. exn:: The / modifier may only occur once.
+         .. exn:: To rename arguments the 'rename' flag must be specified.
             :undocumented:
 
-      `&`
-         tells the type checking algorithm to first type check the arguments
-         before the `&` and then to propagate information from that typing context
-         to type check the remaining arguments. See :ref:`bidirectionality_hints`.
+         .. exn:: Flag 'rename' expected to rename @name into @name.
+            :undocumented:
 
-         .. exn:: The & modifier may only occur once.
+         .. exn:: Arguments of section variables such as @name may not be renamed.
             :undocumented:
 
-      :n:`( ... ) {? % @scope }`
-         :n:`(@name__1 @name__2 ...)%@scope` is shorthand for :n:`@name__1%@scope @name__2%@scope ...`
+      :n:`( {+ @argument_spec } ) {* % @scope }`
+         :n:`(@name__1 @name__2 ...){* %@scope }` is shorthand for
+         :n:`@name__1{* %@scope } @name__2{* %@scope } ...`
```

Comment:
I notice the original text has `scope_key` in the grammar but uses `scope` in the descriptions.  Which one is correct? If you're willing, another thing to clean up.

Reply:
You're right. `scope` (that is either `scope_key` or `scope_name`) is the correct one according to that sentence:

> :token:`scope` can be either a scope name or its delimiting key

(and the code indeed accepts both), the discrepancy in the grammar is fixed by https://github.com/coq/coq/pull/16518